nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -129,16 +128,14 @@ def __init__(
         """
         :param tensor_processor: Backend-specific tensor processor.
         :param aggregation_axes: Axes along which to operate.
-            Registered statistics are stacked along zero axis,
-            axes >=1 correspond to recieved statistic axes shifted left by 1.
         :param num_samples: Maximum number of samples to collect. Aggregator
             skips tensor registration if tensor registration was called num_samples times before.
             Aggregator never skips registration if num_samples is None.
         """
 
         self._tensor_processor = tensor_processor
-        self._aggregation_axes = (0,) if aggregation_axes is None else aggregation_axes
-        self._keepdims = False
+        self._aggregation_axes = (0,) if aggregation_axes is None else (0, *map(lambda x: x + 1, aggregation_axes))
```

Comment:
This is working, no doubts, but I would prefer to avoid labmdas by sacrificing python ninja oneliner